docs/pressable.md

```diff
@@ -0,0 +1,175 @@
+---
+id: pressable
+title: Pressable
+---
+
+Pressable is a primitive core component wrapper that is used to detect various types of interaction states on any of it's defined children.
+
+```jsx
+<Pressable onPress={onPressFunction}>
+  <Text>I'm pressable!</Text>
+</Pressable>
+```
+
+## Example
+
+```
+import * as React from 'react';
+import {
+  Pressable,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+
+export default function App() {
+  const [timesPressed, setTimesPressed] = useState(0);
+
+  let textLog = '';
+  if (timesPressed > 1) {
+    textLog = timesPressed + 'x onPress';
+  } else if (timesPressed > 0) {
+    textLog = 'onPress';
+  }
+
+  return (
+    <View>
+      <Pressable

```

Comment:
I think the example should change which style it uses based on press state too, since that is one of the key parts of Pressable

Reply:
Agreed. Is this possible?

Reply:
I can do it with inline conditionals - @TheSavior were you thinking of a method to do this in particular or is any approach fine for this case?

Reply:
Inline conditional is what I had in mind! 🙂